DQMOffline/Trigger/python/BTaggingMonitor_cfi.py

```diff
@@ -2,114 +2,108 @@
-hltBTVmonitoring.histoPSet.jetEtaBinning = cms.vdouble(-2.4,-2.1,-1.5,-0.9,-0.3,0.,0.3,0.9,1.5,2.1,2.4)
-hltBTVmonitoring.histoPSet.muEtaBinning  = cms.vdouble(-2.4,-2.1,-1.5,-0.9,-0.3,0.,0.3,0.9,1.5,2.1,2.4)
-#pt binning
-hltBTVmonitoring.histoPSet.elePtBinning = cms.vdouble(0,3,5,8,15,20,25,30,40,50,60,80,120,200,400,700)
-hltBTVmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,3,5,8,15,20,25,30,40,50,70,100,150,200,400,700,1000,1500,3000)
-hltBTVmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,3,5,7,10,15,20,30,40,50,70,100,150,200,400,700)
-#Eta binning 2D
-hltBTVmonitoring.histoPSet.eleEtaBinning2D = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
-hltBTVmonitoring.histoPSet.jetEtaBinning2D = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
-hltBTVmonitoring.histoPSet.muEtaBinning2D  = cms.vdouble(-2.5,-1.5,-0.6,0.,0.6,1.5,2.5)
-#pt binning 2D
-hltBTVmonitoring.histoPSet.elePtBinning2D = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
-hltBTVmonitoring.histoPSet.jetPtBinning2D = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
-hltBTVmonitoring.histoPSet.muPtBinning2D  = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
-#HT and phi binning 2D
-hltBTVmonitoring.histoPSet.HTBinning2D  = cms.vdouble(0,20,40,70,100,150,200,400,700)
-hltBTVmonitoring.histoPSet.phiBinning2D = cms.vdouble(-3.1416,-1.8849,-0.6283,0.6283,1.8849,3.1416)
+hltBTVmonitoring = topMonitoring.clone(
+    FolderName = 'HLT/BTV/default/',
+    
+    histoPSet = dict(
+         lsPSet = dict(
+                nbins = 250,
+                xmin  =   0.,
+                xmax  =  2500.),
+         metPSet = dict(
+                nbins =  30,
+                xmin  =   0,
+                xmax  = 300),
+        #lsPSet = dict(                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
+            #   nbins =  2500 ),                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
```

Comment:
This commented line is causing differences in binning.
Please set above:
 lsPSet = dict(
                nbins = 2500,
                xmin  =   0.,
                xmax  =  2500.),
to keep the old behaviour

Reply:
Okay sure. I commented those lines as it was repeated. Let me do the change.